test/integration/enterprises.js

```diff
@@ -28,7 +28,7 @@ describe('(/enterprises) Enterprises API', () => {
     phone : '00904940950932016',
     location_id : '1f162a10-9f67-4788-9eff-c1fea42fcc9b',
     currency_id : 1,
-    settings : { enable_price_lock : 0 },
+    settings : { enable_price_lock : 0, month_average_consumption : 10 },
```

Comment:
Can you check that the GET request returns this value?

Can you also write a test that shows it properly sets the default value?